### PR TITLE
Imp/winlogbeat parsing fields

### PR DESCRIPTION
**Pull request by tfanouillere-sekoia**
## Type of Change

- [ ] New intake format
- [ ] Update to existing intake format
- [ ] Bug fix
- [ ] Documentation update
- [ ] Other (please describe):

## Affected Intake Formats

- Vendor/product:

### 🔒 CRITICAL - Security Requirements (MUST BE VERIFIED FIRST)

- [ ] **No real email addresses** in test files (use: user@example.com, test@example.org)
- [ ] **No real passwords or API keys** (use: "P@ssw0rd!", "xxxxxxxxxxxx", "FAKE_TOKEN_123")
- [ ] **No real IP addresses** (use TEST-NET ranges: 192.0.2.x, 198.51.100.x, 203.0.113.x)
- [ ] **No real phone numbers** (use: +1-555-0100 or similar test numbers)
- [ ] **No real names or PII** (use: "John Doe", "Jane Smith", "User123")
- [ ] **No real company/organization names** in test data (unless public vendors)
- [ ] **No real usernames, hostnames, or domain names** (use: user1, host.example.com, test.corp)
- [ ] All test data has been anonymized and verified

### Required for All PRs

- [ ] Code has been linted
    - [ ] with the linter for manifest, smart-descriptions, and test files (`uv run python linter.py check --changes` in utils/)
    - [ ] with Prettier for the parser code files (`npx prettier --write <module>/<format>/ingest/parser.yml`)
- [ ] All CI/CD checks pass
- [ ] Changes follow the contribution guidelines in `CONTRIBUTING.md`

### Required for New/Updated Intake Formats

- [ ] Clear descriptions provided for modules and formats
- [ ] Logo files included for new modules/formats
- [ ] Parser test coverage is at least 75%
- [ ] At least one smart-description is provided
- [ ] README.md updated (if applicable)
- [ ] Taxonomy mappings are correct and documented
- [ ] Test cases cover edge cases and error handling

### Testing

- [ ] Local tests pass (`uv run pytest` in utils/)
- [ ] Sample data is representative and **completely anonymized**
- [ ] **No sensitive information** (PII, credentials, real IPs, real emails) in test files
- [ ] Test data uses example.com/example.org domains and TEST-NET IP ranges
- [ ] Parser handles malformed data gracefully

### Documentation

- [ ] Code changes are documented
- [ ] Smart-descriptions are clear and informative
- [ ] Field mappings are explained

## Related Issues

Closes #

## Summary by Sourcery

Enhance Winlogbeat event parsing for security and process monitoring use cases.

New Features:
- Parse TaskContent XML from Windows event logs to enrich process and task-related fields.
- Map additional Defender and task-related attributes into action.properties and process/network fields for richer telemetry.
- Add new test fixtures covering various security, process, registry, network, and scheduled task scenarios related to issue 9295.

Enhancements:
- Refine process, parent process, DLL, and destination network field mappings to better normalize Windows event data.
- Remove the unused event.origin field from the Winlogbeat field definitions.

### CODE REVIEW

**Review by mchupeau-sk — APPROVED** [review]
LGFM

**Review by mchupeau-sk — CHANGES_REQUESTED** [review]
Need to be review, after the merging of "uv"

File: Beats/winlogbeat/ingest/parser.yml [hunk 1]
```diff
@@ -692,7 +709,17 @@ stages:
...
+          action.properties.ThreatName: "{{json.event.winlog.event_data.get('Threat Name')}}"
+          action.properties.TaskContentNew_Args: "{{parsed_task_content_xml.parsed_task.Task.Actions.Exec.Arguments}}"
+          action.properties.TaskContentNew_Command: "{{parsed_task_content_xml.parsed_task.Task.Actions.Exec.Command}}"
+          destination.ip: "{{json.event.winlog.event_data.DestinationIp or json.event.winlog.event_data.DestAddress}}"
+          destination.port: "{{(json.event.winlog.event_data.DestinationPort or json.event.winlog.event_data.DestPort) | int}}"
+          dll.path: "{{json.event.winlog.event_data.ImageLoaded}}"
+          dll.name: "{{json.event.winlog.event_data.OriginalFileName}}"
+          process.command_line: "{{json.event.winlog.event_data.CommandLine or (parsed_task_content_xml.parsed_task.Task.Actions.Exec.Command ~ ' ' ~ parsed_task_content_xml.parsed_task.Task.Actions.Exec.Arguments if parsed_task_content_xml.parsed_task.Task.Actions.Exec.Command else None)}}"
+          process.args: "{{json.event.winlog.event_data.CommandLine.split(' ') if json.event.winlog.event_data.CommandLine else None}}"
```
[the remark this answers is not in the document]
Comment (squioc) [reply Beats/winlogbeat/ingest/parser.yml:722]: Hello, I agree with the Copilot's comment.
Can you remove this line?
Thank you

File: Beats/winlogbeat/ingest/parser.yml [hunk 2]
```diff
@@ -692,7 +709,17 @@ stages:
...
-          process.command_line: "{{json.event.winlog.event_data.CommandLine}}"
+          process.parent.command_line: "{{json.event.winlog.event_data.ParentCommandLine}}"
+          action.properties.MessEventType: "{{json.event.winlog.event_data.EventType}}"
+          action.properties.ThreatName: "{{json.event.winlog.event_data.get('Threat Name')}}"
+          action.properties.TaskContentNew_Args: "{{parsed_task_content_xml.parsed_task.Task.Actions.Exec.Arguments}}"
+          action.properties.TaskContentNew_Command: "{{parsed_task_content_xml.parsed_task.Task.Actions.Exec.Command}}"
+          destination.ip: "{{json.event.winlog.event_data.DestinationIp or json.event.winlog.event_data.DestAddress}}"
+          destination.port: "{{(json.event.winlog.event_data.DestinationPort or json.event.winlog.event_data.DestPort) | int}}"
+          dll.path: "{{json.event.winlog.event_data.ImageLoaded}}"
+          dll.name: "{{json.event.winlog.event_data.OriginalFileName}}"
```
Comment (squioc) [inline Beats/winlogbeat/ingest/parser.yml:720]: For the original file name  [`dll.pe.original_file_name`](https://www.elastic.co/docs/reference/ecs/ecs-pe#field-pe-original-file-name) is more suitable. For `dll.name`, the base name of the loaded image is recommended:

```suggestion
          dll.name: "{{json.event.winlog.event_data.ImageLoaded | basename}}"
          dll.pe.original_filen_name: "{{json.event.winlog.event_data.OriginalFileName}}"
```

File: Beats/winlogbeat/ingest/parser.yml [hunk 3]
```diff
@@ -70,7 +77,17 @@ stages:
...
```
Comment (squioc) [inline Beats/winlogbeat/ingest/parser.yml:88]: For the original file name  [`dll.pe.original_file_name`](https://www.elastic.co/docs/reference/ecs/ecs-pe#field-pe-original-file-name) is more suitable. For `dll.name`, the base name of the loaded image is recommended:

```suggestion
          dll.name: "{{json.event.winlog.event_data.ImageLoaded | basename}}"
          dll.pe.original_filen_name: "{{json.event.winlog.event_data.OriginalFileName}}"
```

File: Beats/winlogbeat/tests/test_issue_9295_defender_1117.json
Comment (squioc) [inline Beats/winlogbeat/tests/test_issue_9295_defender_1117.json:1]: Can you redact UUIDs and ip addresses?

File: Beats/winlogbeat/tests/test_issue_9295_credentials_extraction_1.json
Comment (squioc) [inline Beats/winlogbeat/tests/test_issue_9295_credentials_extraction_1.json:1]: Can you redact UUIDs and ip addresses?

File: Beats/winlogbeat/tests/test_issue_9295_dll_side_loading.json
Comment (squioc) [inline Beats/winlogbeat/tests/test_issue_9295_dll_side_loading.json:1]: Can you redact UUIDs and ip addresses?

File: Beats/winlogbeat/tests/test_issue_9295_dns_query_22.json
Comment (squioc) [inline Beats/winlogbeat/tests/test_issue_9295_dns_query_22.json:1]: Can you redact UUIDs and ip addresses?

File: Beats/winlogbeat/tests/test_issue_9295_image_load_dbghelp.json
Comment (squioc) [inline Beats/winlogbeat/tests/test_issue_9295_image_load_dbghelp.json:1]: Can you redact UUIDs and ip addresses?

File: Beats/winlogbeat/tests/test_issue_9295_mshta_process.json
Comment (squioc) [inline Beats/winlogbeat/tests/test_issue_9295_mshta_process.json:1]: Can you redact UUIDs and ip addresses?

File: Beats/winlogbeat/tests/test_issue_9295_mshta_scheduled_task.json
Comment (squioc) [inline Beats/winlogbeat/tests/test_issue_9295_mshta_scheduled_task.json:1]: Can you redact UUIDs and ip addresses?

File: Beats/winlogbeat/tests/test_issue_9295_mssql_shell.json
Comment (squioc) [inline Beats/winlogbeat/tests/test_issue_9295_mssql_shell.json:1]: Can you redact UUIDs and ip addresses?

File: Beats/winlogbeat/tests/test_issue_9295_outbound_kerberos_5156.json
Comment (squioc) [inline Beats/winlogbeat/tests/test_issue_9295_outbound_kerberos_5156.json:1]: Can you redact UUIDs and ip addresses?

File: Beats/winlogbeat/tests/test_issue_9295_powershell_web_request.json
Comment (squioc) [inline Beats/winlogbeat/tests/test_issue_9295_powershell_web_request.json:1]: Can you redact UUIDs and ip addresses?

File: Beats/winlogbeat/tests/test_issue_9295_powershell_encoding.json
Comment (squioc) [inline Beats/winlogbeat/tests/test_issue_9295_powershell_encoding.json:1]: Can you redact UUIDs and ip addresses?

File: Beats/winlogbeat/tests/test_issue_9295_registry_13.json
Comment (squioc) [inline Beats/winlogbeat/tests/test_issue_9295_registry_13.json:1]: Can you redact UUIDs and ip addresses?

File: Beats/winlogbeat/tests/test_issue_9295_remote_thread_8.json
Comment (squioc) [inline Beats/winlogbeat/tests/test_issue_9295_remote_thread_8.json:1]: Can you redact UUIDs and ip addresses?

File: Beats/winlogbeat/tests/test_issue_9295_sliver_dns_22.json
Comment (squioc) [inline Beats/winlogbeat/tests/test_issue_9295_sliver_dns_22.json:1]: Can you redact UUIDs and ip addresses?

File: Beats/winlogbeat/tests/test_issue_9295_scheduled_task_4698.json
Comment (squioc) [inline Beats/winlogbeat/tests/test_issue_9295_scheduled_task_4698.json:1]: Can you redact UUIDs and ip addresses?

File: Beats/winlogbeat/tests/test_issue_9295_sysmon_4769_tgs.json
Comment (squioc) [inline Beats/winlogbeat/tests/test_issue_9295_sysmon_4769_tgs.json:1]: Can you redact UUIDs and ip addresses?

**Review by squioc — CHANGES_REQUESTED** [review]
Thank you for the work.

Some changes to apply.

**Comment by TOUFIKIzakarya** [comment]
Hello,

This PR was closed, because we have [another one](https://github.com/SEKOIA-IO/intake-formats/pull/2528/changes) that resolved all comments.
